tests/block_atx/remove_node/test_blocks_remove_node.py

```diff
@@ -0,0 +1,145 @@
+import time
+
+from pytest_testconfig import config as testconfig
+
+from tests import queries as q
+from tests.deployment import delete_deployment
+from tests.test_bs import setup_network, add_curl, setup_bootstrap, start_poet, setup_clients, wait_genesis, get_conf
+from tests.test_bs import add_multi_clients
+from tests.utils import validate_blocks_per_nodes, get_pod_id
+
+
+# epoch i:
+# start with x miners
+# add another single miner
+# wait 3 epochs
+#
+# epoch i+3:
+# validate total miners generated Tavg/x+1 (floored) in i+1 and i+2
+# validate all miners created an ATX
+# remove the last added miner
+#
+# epoch i+4:
+# validate total miner generated Tavg/x+1 (floored) in i+3 except for the removed miner
```

Comment:
why are we validating this for 3 epochs? this will make test run longer

Reply:
changing it

Reply:
fixed in 9f8631cc52a2c63169d19fb61326af61d5adee98